src/app/banking/vouchers/view-voucher.component.html

```diff
@@ -0,0 +1,89 @@
+<page-layout [ready]="data$ | async">
+  <page-content *ngIf="data$ | async; else loading" [heading]="data.type.name"
+    [mobileHeading]="data.type.name"
+    [headingActions]="headingActions$ | async">
+
+    <div class="d-flex flex-grow-1 flex-column">
+      <div
+        [class]="(layout.gtsm$ | async) ? 'd-flex flex-row-reverse justify-content-between' : 'd-flex flex-column' ">
+        <div *ngIf="data.type.image"
+          class="image d-flex justify-content-center mt-2 mb-4 align-self-center">
+          <avatar size="160" imageSize="200" useLightbox
+            [image]="data.type.image"></avatar>
+        </div>
+        <div class="d-flex flex-column flex-grow-1">
+          <label-value [label]="i18n.general.type"> {{ data.type.name}}
+          </label-value>
+          <label-value [label]="i18n.voucher.token"> {{ data.token}}
+          </label-value>
+          <label-value [label]="i18n.transaction.amount">
+            {{ data.amount | currency:data.type.configuration.currency }}
+          </label-value>
+          <label-value *ngIf="data.creationType"
+            [label]="i18n.voucher.creationType">
+            {{ data.creationType | capitalizeFirst }}
+          </label-value>
+          <label-value [label]="i18n.general.creationDate">
+            {{ data.creationDate | dateTime}}
+          </label-value>
+          <label-value *ngIf="data.buyer" [label]="i18n.voucher.buy.buyer">
+            <user-link [user]="data.buyer"></user-link>
+          </label-value>
+          <label-value *ngIf="data.buy" [label]="i18n.voucher.buy.payment">
+            <a
+              [routerLink]="['/banking/transfer/' + data.buy.transactionNumber]">
```

Comment:
Either use the array or string form. Also, you cannot rely that transaction number will exist, as it is optional in Cyclos.
It is maybe helper to create a `get buyPath(): string` in the ts, and do something like `AccountHistoryComponent.pathP .